docs/trex_release-notes.md

```diff
@@ -72,6 +72,38 @@ Bugs fixed in this release:
 
 * The documentation for the <a href="{{site.baseurl}}/docs/interfaces/worksheet.html#selectmarksbyvalueasync" target="_blank"><code>selectMarksByValueAsync</code></a> method has been corrected. If you are calling the method, be sure to specify the complete namespace for the `SelectionUpdateType` enum that is passed to the method as the `updateType` parameter.  For example, use `tableau.SelectionUpdateType.Replace`, to replace the currently selected marks with the values you specify in the method call.
 
+----
+### Extensions API library v1.2
```

Comment:
Any reason I don't see the release notes for v1.3 ?

Reply:
Dave already merged it into dev.